### ISSUE TITLE
Adding holidays for Portugal 

### ISSUE THREAD

**Issue by fstueber** [openpotato/openholidaysapi.data#26]
Initial data provided by @48design:
https://github.com/openpotato/openholidaysapi.data/tree/feature/pt/src/pt

**Comment by fstueber** [comment]
Question to @48design:
- According to [Wikipedia](https://pt.wikipedia.org/wiki/Feriados_em_Portugal) municipal holidays in Portugal are optional, that is, they can be observed by companies but not necessarily. Should we really put them as public holidyays to the database?

**Comment by 48design** [comment]
We could either set the Type column to Optional, or put them in a new file together, like `holidays.optional.csv`?

**Comment by fstueber** [comment]
Yes, maybe. They definitely need another type. Will think about that. There is a an issue #10  for adding non-statutory holidays, which is similar
